openproblems/tasks/multimodal_data_integration/__init__.py

```diff
@@ -5,7 +5,9 @@
 from . import metrics
 
 _task_name = "Multimodal Data Integration"
-_task_summary = "Alignment of single-cell data from two different modalities"
+_task_summary = (
+    "Jointly embedding single-cell omics datasets across modalities and batches"
+)
```

Comment:
I misunderstood the task when I initially wrote this blurb. Maybe the original was more accurate. I would adapt to:
> Alignment of cellular profiles from two different modalities